xinference/tests/test_client.py

```diff
@@ -75,6 +75,28 @@ def test_client_for_embedding(setup):
     assert len(client.list_models()) == 0
 
 
+def test_replica_model(setup):
+    endpoint, _ = setup
+    client = Client(endpoint)
+    assert len(client.list_models()) == 0
+
+    model_uid = client.launch_model(
+        model_name="orca", model_size_in_billions=3, quantization="q4_0", replica=2
+    )
+    assert len(client.list_models()) == 2
```

Comment:
I think users are not supposed to see the replica model uid. Thus, list_models should return only one item.

Reply:
Fixed, thanks.